mob.go

```diff
@@ -877,10 +867,10 @@ func (branch Branch) isOrphanWipBranch(configuration Configuration) bool {
 }
 
 func branch(configuration Configuration) {
-	say(silentgit("branch", "--list", "--remote", newBranch("*").addWipPrefix(configuration).remote(configuration).Name))
+	say.Say(silentgit("branch", "--list", "--remote", newBranch("*").addWipPrefix(configuration).remote(configuration).Name))
```

Comment:
does that make sense? why don't we use `git` instead of `say.Say(silentgit(...))`